### ISSUE TITLE
feat(datepicker): add inline visualization option

### ISSUE THREAD

**Issue by gustavolira** [valor-software/ngx-bootstrap#461]
Would be nice if was possible to configure datepicker to be displayed inline

**Comment by valorkin** [comment]
sample please

**Comment by gustavolira** [comment]
https://jqueryui.com/datepicker/
Datepicker is only open when user click on the input

Thank's for your attention

**Comment by johanchouquet** [comment]
What I did : On click on the calendar Icon, i show the datepicker component as a toggle.

_Template:_
`<i class="glyphicon glyphicon-calendar" (click)="toggleDP()"></i><br/>`

_Component:_

```
public clicked:boolean = false;
public toggleDP():boolean {
    this.clicked = !this.clicked;
    return !this.clicked;
  }
```

It's not exactly the same as in JQuery UI, but it's with the same intent.

**Comment by CarlCui-AI** [comment]
Many easy workarounds. I simply use a ngIf for toggling.

**Comment by alfakappa** [comment]
Yes, there are many workarounds, but original Bootstrap accounts for a datepicker-inline attribute. I think TS is referring to this.

**Comment by valorkin** [comment]
available in new datepicker from v1.9.2
http://valor-software.com/ngx-bootstrap/#/datepicker
